metaflow/task.py

```diff
@@ -291,25 +295,27 @@ def clone_only(
                 "task.clone_only needs a valid clone_origin_task value."
             )
         origin_run_id, _, origin_task_id = clone_origin_task.split("/")
-
-        msg = {
-            "task_id": task_id,
-            "msg": "Cloning task from {}/{}/{}/{} to {}/{}/{}/{}".format(
-                self.flow.name,
-                origin_run_id,
-                step_name,
-                origin_task_id,
-                self.flow.name,
-                run_id,
-                step_name,
-                task_id,
-            ),
-            "step_name": step_name,
-            "run_id": run_id,
-            "flow_name": self.flow.name,
-            "ts": round(time.time()),
-        }
-        self.event_logger.log(msg)
+        msg = "Cloning task from {}/{}/{}/{} to {}/{}/{}/{}".format(
+            self.flow.name,
+            origin_run_id,
+            step_name,
+            origin_task_id,
+            self.flow.name,
+            run_id,
+            step_name,
+            task_id,
+        )
+        with _system_monitor.count("metaflow.task.clone"):
+            self.event_logger.log(
+                {
```

Comment:
The log message format seems specific to the logging consumer. If that is the case, consider keeping `self.event_logger.log(msg)` as is to maintain flexibility and decorate the payload with `event_value`, `log_type`and `stream_type` inside an `atlas_logger` class.

Reply:
Will update the API in the revised PR.